azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerYamlTest.java

```diff
@@ -56,6 +57,7 @@ public void testBasicFlowWithoutEndNode() throws Exception {
     assertFlowStatus(flow, Status.SUCCEEDED);
   }
 
+  @Ignore
```

Comment:
Could include a comment, something like this:

There seems to be an actual race condition bug in the runtime code. See: issue #1921: Flaky test FlowRunnerTestYaml & issue #1311: Potential race condition between flowRunner thread and jetty killing thread

Reply:
Good suggestion! I will update it.